### PR TITLE
Add usage example to Dropout

### PR DESCRIPTION
**Pull request by Joseph-Rance**
Added a usage example to tf.keras.layers.Dropout

### CODE REVIEW

**Comment by Joseph-Rance** [comment]
@googlebot I signed it!

File: tensorflow/python/keras/layers/core.py [hunk 1]
```diff
@@ -145,6 +145,12 @@ class Dropout(Layer):
     inputs: Input tensor (of any rank).
     training: Python boolean indicating whether the layer should behave in
       training mode (adding dropout) or in inference mode (doing nothing).
+      
+  Usage Example:
+    >>> import tensorflow as tf
+    >>> model = tf.keras.models.Sequential()
+    >>> model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
+    >>> model.add(tf.keras.layers.Dense(1))
```
Comment (rthadur) [inline tensorflow/python/keras/layers/core.py]: Thanks for your contribution , can you please remove chevron(>>>) symbol . 
Here is the link for [contribution guidelines.](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#382-modules)

**Review by rthadur — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/layers/core.py (hunk 1 above)
Comment (Joseph-Rance) [reply tensorflow/python/keras/layers/core.py]: Hi. I think I have removed them all.

**Comment by boronhub** [comment]
Please move the Usage Examples section above the Arguments section

**Comment by Joseph-Rance** [comment]
Hi, I've moved it above the Arguments section.

File: tensorflow/python/keras/layers/core.py [hunk 2]
```diff
@@ -130,6 +130,12 @@ class Dropout(Layer):
   Dropout consists in randomly setting
   a fraction `rate` of input units to 0 at each update during training time,
   which helps prevent overfitting.
+  
+  Usage Example:
+     import tensorflow as tf
```
Comment (RichardXiao13) [inline tensorflow/python/keras/layers/core.py]: You can get rid of this import statement.

**Comment by RichardXiao13** [comment]
@Joseph-Rance Be sure to make the usage example usable with the doctest by making sure every line of code begins with `>>> `. Then, make sure to provide the output below the code between `< >`.

File: tensorflow/python/keras/layers/core.py (hunk 2 above)
Comment (Joseph-Rance) [reply tensorflow/python/keras/layers/core.py]: Thanks , @RichardXiao13. I've removed it.

**Comment by gbaned** [comment]
@Joseph-Rance Can you please resolve conflicts? Thanks!

**Comment by Joseph-Rance** [comment]
Hi, they should be fixed now

File: tensorflow/python/keras/layers/core.py [hunk 3]
```diff
@@ -140,6 +140,11 @@ class Dropout(Layer):
   (This is in contrast to setting `trainable=False` for a Dropout layer.
   `trainable` does not affect the layer's behavior, as Dropout does
   not have any variables/weights that can be frozen during training.)
+  
+   Usage Example:
+   model = tf.keras.models.Sequential()
+   model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
+   model.add(tf.keras.layers.Dense(1))
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/core.py]: This doesn't follow the proper syntax at https://www.tensorflow.org/community/contribute/docs. Should be

```suggestion
   Usage Example:
   
   >>> model = tf.keras.models.Sequential()
   >>> model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
   >>> model.add(tf.keras.layers.Dense(1))
```

File: tensorflow/python/keras/layers/core.py [hunk 4]
```diff
@@ -158,7 +163,7 @@ class Dropout(Layer):
    [ 5.    6.25]
    [ 7.5   8.75]
    [10.    0.  ]], shape=(5, 2), dtype=float32)
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/core.py]: Trailing whitespace. Should be
```suggestion

```

File: tensorflow/python/keras/layers/core.py [hunk 5]
```diff
@@ -158,7 +163,7 @@ class Dropout(Layer):
    [ 5.    6.25]
    [ 7.5   8.75]
    [10.    0.  ]], shape=(5, 2), dtype=float32)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/core.py]: We already have a usage example

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
There is already one usage example. What does the new example bring?

File: tensorflow/python/keras/layers/core.py (hunk 5 above)
Comment (Joseph-Rance) [reply tensorflow/python/keras/layers/core.py]: I added it because I thought it would be simpler to understand if an example with a sequential model was also used

Comment (mihaimaruseac) [reply tensorflow/python/keras/layers/core.py]: Yes, but the addition is not testable and is already included in all tutorials using Keras layers.

Comment (Joseph-Rance) [reply tensorflow/python/keras/layers/core.py]: Ok, fair enough. Should I close the PR?

Comment (mihaimaruseac) [reply tensorflow/python/keras/layers/core.py]: I think that's the good plan forward from here. Thank you for the contribution, let's try to get others
